### DISCUSSION TITLE
Wind gust confusion

### DISCUSSION THREAD

**Discussion by jfbauer432** [weather-gov/api#352] [Q&A] [answered]
I know that most wind reports do not include gust unless they exceed some threshold.  Looking at the gridpoints (e.g. https://api.weather.gov/gridpoints/LWX/89,86) data there are values for windSpeed and windGust.  The windGust values seem to always exist.

Comparing that to Hourly graphs like https://forecast.weather.gov/MapClick.php?lat=39.264&lon=-77.175&unit=0&lg=english&FcstType=graphical , the gusts are only shown in certain circumstances.  And the rules for that appears to be https://graphical.weather.gov/definitions/defineWindGust.html

So I applied those rule to the wind/windgusts in the gridpoints data.  But that eliminates a lot more gusts then the above Hourly graph shows.  And yes, I did adjust the units to match (converting but m/s and knots to mph).

What am I not understanding here?

**Comment by StephenClouse** [comment, marked as answer]
The gridpoints endpoint is the raw data we receive from the field offices, unaltered other than serializing it from NetCDF to JSON. We do not alter or attempt to cull any data here.

It is hard to make any comparisons to the current forecast site because it operates on its own business rules that no one really bothered to write down.

In any case, there is no specific issue that I can discern here with the API. If the question is about the forecast site, that is not something we can support via this forum.
